frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeaderButtonRow/DashboardHeaderButtonRow.unit.spec.tsx

```diff
@@ -42,7 +42,7 @@ const DASHBOARD_EXPECTED_DATA_MAP: Record<
   },
   [DASHBOARD_ACTION.ADD_LINK_CARD]: {
     icon: "link",
-    tooltip: "Add link card",
+    tooltip: "Add a link or iFrame",
```

Comment:
This spelling is incorrect (it's not an apple product :smile:) . It should be either `IFrame` or `iframe`.
MDN uses `IFrame` ([example](https://developer.mozilla.org/en-US/docs/Web/Security/IFrame_credentialless)) and `iframe` ([example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#allow)).
Wikipedia uses `iframe` ([example](https://en.wikipedia.org/wiki/HTML_element#Frames)).
W3C uses `iframe` ([example](https://wicg.github.io/anonymous-iframe/))

IMO: `iframe` is most natural for people who know what an iframe is.
It should be `Iframe` if it's a first word in a sentence.

----

This PR uses both `iFrame` and `IFrame`. Let's pick one and make it consistent throughout.